### ISSUE TITLE
[BUG] [FORMATTER]

### ISSUE THREAD

**Issue by jgillard** [Riverside-Healthcare/djLint#554]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: MacOS 13.2
 - Python 3.11.2
 - djlint, version 1.19.14
 - template language: jinja

## Issue
Missing whitespace before curly braces: `{{ url('foo')}}`

I've found a bug where a lack of whitespace between the end of `url()` and the closing curly braces is allowed through the linter. I have not dug further into this to find out how generalised the issue is. See below for the 4 variations of whitespace for this example. 1 is correct, 2 are correctly identified, and this example is missed. I had a quick look at rules.py but couldn't work out where the issue would be, so am just opening this report for now, but am up for opening an PR if pointed in the right direction.

## How To Reproduce
I ran `djlint djlint-demo.jinja2 --check` for this file below. The last example is not identified as incorrect.

```jinja
{# Correct #}
{{ url('foo') }}

{# Correctly caught #}
{{url('foo') }}
{{url('foo')}}

{# Missed #}
{{ url('foo')}}
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
None

**Comment by christopherpickering** [comment]
Thanks! I see a typo in the code for the regex for that... I'll let out a fix shortly.
